### PR TITLE
Add YAML and TSV examples to plot show

### PR DESCRIPTION
**Pull request by imhardikj**
Addresses  https://github.com/iterative/dvc.org/issues/1414#issuecomment-729988972:

>Just add an expandable section under the sample data of https://dvc.org/doc/command-reference/plots/show#example-hierarchical-data to show how it looks in other formats.

### CODE REVIEW

File: content/docs/command-reference/plots/show.md [hunk 1]
```diff
@@ -105,6 +105,36 @@ file:///Users/usr/src/plots/train.json.html
 
 > Note that only the last field name (`loss`) is used for the plot by default.
 
+<details>
+
+### Expand to see example using YAML metrics file
```
Comment (jorgeorpinel) [inline content/docs/command-reference/plots/show.md:110]: Let's move this up right after the JSON code block and call it **Expand for YAML format**, please.

File: content/docs/command-reference/plots/show.md [hunk 2]
```diff
@@ -105,6 +105,36 @@ file:///Users/usr/src/plots/train.json.html
...
+
+We'll use metrics file `train.yaml` for this example:
```
Comment (jorgeorpinel) [inline content/docs/command-reference/plots/show.md:112]:
```suggestion
Here's a corresponding `train.yaml` metrics file:
```

File: content/docs/command-reference/plots/show.md [hunk 3]
```diff
@@ -105,6 +105,36 @@ file:///Users/usr/src/plots/train.json.html
...
+
+DVC identifies and plots YAML objects from the first YAML element found in the
+file (`train`):
+
+```dvc
+$ dvc plots show train.yaml
+file:///Users/usr/src/plots/train.yaml.html
+```
+
+![](/img/plots_show_json.svg)
```
Comment (jorgeorpinel) [inline content/docs/command-reference/plots/show.md:134]: No need to repeat the whole example.
```suggestion
```

File: content/docs/command-reference/plots/show.md [hunk 4]
```diff
@@ -139,6 +169,35 @@ file:///Users/usr/src/plots/logs.csv.html
 
 ![](/img/plots_show.svg)
 
+<details>
+
+### Expand to see example using TSV metrics file
```
Comment (jorgeorpinel) [inline content/docs/command-reference/plots/show.md:174]: Please move this right after the YAML section and call it **Expand for TSV format**.

File: content/docs/command-reference/plots/show.md [hunk 5]
```diff
@@ -139,6 +169,35 @@ file:///Users/usr/src/plots/logs.csv.html
...
+
+We'll use tabular metrics file `logs.tsv` for these examples:
```
Comment (jorgeorpinel) [inline content/docs/command-reference/plots/show.md:176]:
```suggestion
Here's a corresponding `train.tsv` metrics file:
```

File: content/docs/command-reference/plots/show.md [hunk 6]
```diff
@@ -139,6 +169,35 @@ file:///Users/usr/src/plots/logs.csv.html
...
+```
+
+By default, this command plots the last column of the table (see `-y` option):
+
+```dvc
+$ dvc plots show logs.tsv
+file:///Users/usr/src/plots/logs.csv.html
+```
+
+![](/img/plots_show.svg)
```
Comment (jorgeorpinel) [inline content/docs/command-reference/plots/show.md:197]: I see you used the next example to mix it up a little, good thinking but the task was simpler than this:

> Just add an expandable section under the sample data ... to show how it looks in other formats.

```suggestion
```

File: content/docs/command-reference/plots/show.md (hunk 1 above)
Comment (imhardikj) [reply content/docs/command-reference/plots/show.md]: Updated

File: content/docs/command-reference/plots/show.md (hunk 4 above)
Comment (imhardikj) [reply content/docs/command-reference/plots/show.md]: Done

File: content/docs/command-reference/plots/show.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/plots/show.md]: It's still not right after the JSON code block.

File: content/docs/command-reference/plots/show.md [hunk 7]
```diff
@@ -137,6 +157,26 @@ $ dvc plots show logs.csv
 file:///Users/usr/src/plots/logs.csv.html
 ```
 
+<details>
+
+### Expand for TSV format
+
+Here's a corresponding `train.tsv` metrics file:
```
Comment (jorgeorpinel) [inline content/docs/command-reference/plots/show.md:164]: Same here, just put both these blocks right after the JSON block code please...

**Review by jorgeorpinel — CHANGES_REQUESTED** [review]
Again, pleass move JSON/TSV blocks right after JSON block.

File: content/docs/command-reference/plots/show.md (hunk 7 above)
Comment (imhardikj) [reply content/docs/command-reference/plots/show.md]: Done. But the TSV block would look better after CSV block code think.

Comment (jorgeorpinel) [reply content/docs/command-reference/plots/show.md]: > TSV block would look better after CSV block

Wait that does sound logical, let me check... ⌛

File: content/docs/command-reference/plots/show.md [hunk 8]
```diff
@@ -93,6 +93,46 @@ We'll use tabular metrics file `train.json` for this example:
...
+1    0.9763333    0.07896138601688048     0.9768    0.07310650711813942
+2    0.98375      0.05241111190887168     0.9788    0.06665669009438716
+3    0.988016     0.03681169906261687     0.9781    0.06697812260198989
+4    0.991116     0.027362171787042946    0.978     0.07385754839298315
+5    0.9932333    0.02069501801203781     0.9771    0.08009233058886166
+6    0.9945       0.017702101902437668    0.9803    0.07830339228538505
+7    0.9954       0.01396906608727198     0.9802    0.07247738889862157
+```
+
+</details>
```
Comment (jorgeorpinel) [inline content/docs/command-reference/plots/show.md:134]: You're right @imhardikj, this should be in the next example 🤦 Sorry, my bad.

**Review by jorgeorpinel — APPROVED** [review]
Please move it down and I'll merge this.

**Comment by jorgeorpinel** [comment]
Thanks again @imhardikj !

**Pull request merged**